js/script.js

```diff
@@ -0,0 +1,47 @@
+const link = document.querySelector(".main-form-button");
+const popup = document.querySelector(".form-wrap");
+const form = popup.querySelector("form");
+const dateStart = popup.querySelector("[name=datestart]");
+const adult = popup.querySelector("[name=adult]");
+const children = popup.querySelector("[name=children]");
+
+let isStorageSupport = true;
+const storageDateStart = "";
+
+try {
+  storageDateStart = localStorage.getItem("dateStart");
+} catch (err) {
+  isStorageSupport = false;
+}
+
+if (storageDateStart) {
+  dateStart.value = storageDateStart;
+  dateEnd.focus();
+} else {
+  dateStart.focus();
+}
+
+link.addEventListener("click", function () {
```

Comment:
можно использовать стрелочные функции:
```suggestion
link.addEventListener("click", () => {
```